src/_designs/components/generation/index.vue

```diff
@@ -43,7 +40,7 @@
           title="Total"
         >
           <FormatAeUnit
-            :value="data.key_block.total"
+            :value="data.keyBlock.target"
```

Comment:
The data (total, reward) is not available as of now. We should just not show the wrong data in its place. We can add the fields when they become available.
For now, I'd recommend removing 'total', 'rewards' fields altogether.

Reply:
updated